massa-models/src/block_header.rs

```diff
@@ -614,7 +614,7 @@ mod test {
     };
     use massa_signature::{verify_signature_batch, KeyPair};
 
-    // Only for testing purpose
+    // Only for test-exports purpose
```

Comment:
Revert the rename here :)

Reply:
I checked by hand the renaming maybe my eyes were broken.

Reply:
after reading "test" a 1000 times, I'm sure it starts getting blurry